packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -18,6 +18,14 @@ const Notification = ({ data, removeNotificationMessage }) => {
     const destroy = is_closed_by_user => {
         removeNotificationMessage(data);
 
+        if (data.should_show_again) {
+            const closed_toast_notifications = JSON.parse(localStorage.getItem('closed_toast_notifications')) ?? [];
```

Comment:
would it work if we moved `closed_toast_notifications` outside of the function and bring the below one to the top? or does this need to be local to this function only?

Reply:
i need to put it inside of the function because we store the order key to localstorage only after we close the toast notication

Reply:
ahhh alright 👍